### PR TITLE
Add hpo scripts to run an experiment with benchmarks.

### PR DESCRIPTION
**Pull request by kusumachalasani**
Signed-off-by: kusumachalasani <kchalasa@redhat.com>

### CODE REVIEW

**Review by dinogun — CHANGES_REQUESTED** [review]
I see the benchmarks folder in the PR, is this intentional?

**Comment by dinogun** [comment]
@kusumachalasani It looks like these scripts are tied to tfb specifically. How can we abstract it out so that we can plug in for any benchmark ? I think we need to rethink the folder structure from that perspective as well

File: em-hpo-run.sh [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        THREAD="40"
+fi
+
+if [ -z "${DURATION}" ]; then
+        DURATION="60"
+fi
+
+if [ -z "${CONNECTIONS}" ]; then
+        CONNECTIONS="512"
+fi
```
Comment (dinogun) [inline em-hpo-run.sh]: Wondering if these would be better if we just pre initialize them before reading the user parameters and based on the user params only the right ones get set.

```
THREAD="40"
...
case
   ...
        t)
                THREAD="${OPTARG}"
                ;;
...
```

**Comment by dinogun** [comment]
@kusumachalasani Can we clone the autotune repo in the scripts to get access to the HPO scripts just like we do in the `autotune-demo` repo. This way we do not duplicate the code and have a single source for all usecases. After cloning you can checkout tag `v0.2` to get to the same level of HPO code so everything else in this repo can remain the same. You would have to fix up the paths though as HPO will now be inside the autotune folder

**Comment by kusumachalasani** [comment]
> @kusumachalasani Can we clone the autotune repo in the scripts to get access to the HPO scripts just like we do in the `autotune-demo` repo. This way we do not duplicate the code and have a single source for all usecases. After cloning you can checkout tag `v0.2` to get to the same level of HPO code so everything else in this repo can remain the same. You would have to fix up the paths though as HPO will now be inside the autotune folder

But, autotune v0.2 doesn't have all the changes required for the scripts. If we want to use through autotune repo, then we need to update them.

File: em-hpo-run.sh (hunk 1 above)
Comment (kusumachalasani) [reply em-hpo-run.sh]: Done.

**Comment by dinogun** [comment]
Am merging this for now. W need to remove the HPO scripts from here and merge the changes back into the main autotune repo.

**Pull request merged**